src/components/views/ViewDataTable/utils.tsx

```diff
@@ -100,3 +100,49 @@ export function makeGridColDef(
 
   return colDef;
 }
+
+export const viewQuickSearch = (
+  rows: ZetkinViewRow[],
+  columns: ZetkinViewColumn[],
+  quickSearch: string
+): ZetkinViewRow[] => {
+  if (!quickSearch) return rows;
+
+  const getColIdxByFieldType = (fields: string[]) =>
+    columns.reduce((output: number[], input, idx) => {
+      if (fields.includes(input.type)) output.push(idx);
```

Comment:
Like with all if statements, I think this should have a `{}` block instead of a single-line statement. This is what we've previously agreed, but maybe we should add an `eslint` rule for this?